html-css/override-extend-variables/step1-instructions.md

```diff
@@ -35,20 +35,20 @@ PatternFly is an open source design system created to enable consistency and usa
   &lt;/div&gt;
 &lt;/div&gt;
 </pre>
-
-2) <strong>Set up your new styles file for overriding styles in the card component.</strong> There are two blocks in this file: `:root` targets styles across the entire application and `.pf-c-card` targets styles specifically in the card component.
-
-Click the <strong>Copy to Editor</strong> button below to add code for the card component to the myapp.scss file.
-
+ 
+3) <strong>Set up your new styles file for overriding styles in the card component.</strong> There are two blocks in this file: `:root` targets styles across the entire application and `.pf-c-card` targets styles specifically in the card component.
```

Comment:
Add a return after the first sentence, like this:

3) <strong>Set up your new styles file for overriding styles in the card component.</strong> 

There are two blocks in this file: `:root` targets styles across the entire application and `.pf-c-card` targets styles specifically in the card component.